dask_mpi/tests/basic.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from time import sleep
+from distributed import Client
+from distributed.metrics import time
+from tornado import gen
+
+from dask_mpi.core import initialize
+
+scheduler_file = sys.argv[1]
+
+initialize(scheduler_file=scheduler_file)
+
+with Client(scheduler_file=scheduler_file) as c:
+    start = time()
+    while len(c.scheduler_info()['workers']) != 2:
+        assert time() < start + 10
+        sleep(0.2)
+
+    assert c.submit(lambda x: x + 1, 10, workers=1).result() == 11
```

Comment:
Why do you specify a worker kwarg here?

Reply:
See above.  It's what was done in `distributed.cli.tests.test_basic`.  No other reason.

Reply:
To be clear, this is testing that the workers have been named according to rank.  The fact that this doesn't err saying "no worker named 1 exists" is helpful.

Reply:
I should be clear that the workers are named `rank-1` when using `initialize` because `rank==1` is dedicated to the client process.  Do you think this should go back to the `name=rank` convention?  So that the CLI and `initialize` are consistent with each other?

Reply:
Hrm interesting.  I can see value in having the MPI rank and the worker name be the same.  I'm happy to defer to your judement here though @kmpaul .  My guess is that you have more experience and understanding of user expectations with MPI workloads than I do.

Reply:
I think your intuition is correct that having the `{rank}` match the worker name makes sense...though maybe we can make it more explicit, like calling it `rank{rank}` or `MPI{rank}` or the like.  Whatever is clear that the name refers to the MPI rank.

I think, now that I've thought about it for a minute or so, I will implement worker names that match the pattern: `mpi-rank-{rank}`.  That should be explicit enough.

Reply:
Now done in https://github.com/dask/dask-mpi/pull/6/commits/10cc60e33e1ed4fdfff617513e41cf61392038cc